demo/Gemfile.lock

```diff
@@ -108,13 +108,6 @@ GEM
       signet (~> 0.15)
     helm-cli (0.3.0)
       helm-rb (~> 0.1)
-    helm-rb (0.2.1-aarch64-linux)
-    helm-rb (0.2.1-arm-linux)
-    helm-rb (0.2.1-arm64-darwin)
-    helm-rb (0.2.1-arm64-linux)
-    helm-rb (0.2.1-ppc64le-linux)
-    helm-rb (0.2.1-s390x-linux)
-    helm-rb (0.2.1-x86-linux)
```

Comment:
I think this should not be here; when I rebuild the PR locally these changes get undone.